.github/workflows/build_and_test_cvp_image.yml

```diff
@@ -0,0 +1,59 @@
+name: "Build and test CVP CI image"
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  pull_request:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs a set of commands using the runners shell
+      - name: "Build the cvp_image inside Dockerfiles"
+        run: |
+          podman --version
+          cd Dockerfiles 
+          podman build -t cvp_image:latest -f Dockerfile.cvp --build-arg OPERATOR_SDK_VERSION=v1.4.0 --build-arg OPERATOR_TEST_PLAYBOOKS_TAG=master
+
+      - name: "Run example bundle through cvp_image -> POSITVE test"
+        run: |
+          cd Dockerfiles
+          mkdir output_logs
+          podman run -it -v ./example-bundle:/project/operator-bundle -v ./output_logs:/project/output --security-opt label=disable cvp_image:latest
```

Comment:
IIRc ubuntu uses apparmor and not SElinux. And disabling of apparmor shoudl be done with `--security-opt "apparmor=unconfined"`

Reply:
I will verify this.